### PR TITLE
Anim component public api expansion follow up

### PR DESCRIPTION
**Pull request by ellthompson**
Follow up PR for additional comments in #3782

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/framework/components/anim/component-layer.js [hunk 1]
```diff
@@ -299,10 +298,10 @@ class AnimComponentLayer {
     }
 
     /**
-     *  Returns the asset that is associated with the given state.
+     * Returns the asset that is associated with the given state.
      *
      * @param {string} stateName - The name of the state to get the asset for.
-     * @type {pc.Asset}
+     * @returns {pc.Asset} - the asset associated with the given state.
```
Comment (mvaligursky) [inline src/framework/components/anim/component-layer.js:304]: 'the' => 'The' (capital T)

**Review by mvaligursky — APPROVED** [review]

**Review by mvaligursky — APPROVED** [review]

File: src/framework/components/anim/component-layer.js [hunk 2]
```diff
@@ -299,10 +298,10 @@ class AnimComponentLayer {
     }
 
     /**
-     *  Returns the asset that is associated with the given state.
+     * Returns the asset that is associated with the given state.
      *
      * @param {string} stateName - The name of the state to get the asset for.
-     * @type {pc.Asset}
+     * @returns {pc.Asset} - The asset associated with the given state.
```
Comment (willeastcott) [inline src/framework/components/anim/component-layer.js:304]: Remove the `pc.`

Comment (willeastcott) [reply src/framework/components/anim/component-layer.js:304]: Also, remove the `-`

Comment (willeastcott) [reply src/framework/components/anim/component-layer.js:304]: Basically, browse some other `@returns` tags to see how they're done:

https://github.com/playcanvas/engine/blob/dev/src/asset/asset-registry.js#L311

**Review by willeastcott — APPROVED** [review]

**Pull request merged**